wrappers/services.go

```diff
@@ -548,6 +597,11 @@ func (es *ensureSnapServicesContext) reloadModified() error {
 			return err
 		}
 	}
+	if es.journalCtlReloadNeeded {
+		if err := es.sysd.Restart([]string{"systemd-journald.service"}); err != nil {
```

Comment:
Do we need to restart the main journal if AFAIU the affected ones are only sstemd-journald@.service?

Reply:
Thank you so much for pointing this out @bboozzoo ! It turns out you are completely correct and that it's not necessary to restart the journald service at all. The snap service itself only needs restarting (which happens when you set/change the quota), and then the journald service for that is also restarted. (It turns out my research on this topic was flawed, and didn't catch this while testing journal quota)

This was a really good catch!